src/app.rs

```diff
@@ -240,6 +240,22 @@ pub struct App {
     pub hostname: String,
 }
 
+#[cfg(not(target_os = "macos"))]
+fn get_config_dir() -> Option<PathBuf> {
+    dirs::config_dir()
+}
+
+#[cfg(target_os = "macos")]
+fn get_config_dir() -> Option<PathBuf> {
+    match env::var_os("XDG_CONFIG_HOME") {
+        Some(val) => Some(PathBuf::from(val)),
+        None => match dirs::home_dir() {
+            Some(val) => Some(val.join("/Library/Application Support")),
```

Comment:
```suggestion
            Some(val) => Some(val.join(".config")),
```
So we don't break existing config files.